packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(video-background-effects) */
+import { Panel } from '@fluentui/react';
+
+/**
+ * Pane that is used to show video effects button
+ * @private
+ */
+/** @beta */
+export const VideoEffectsPane = (props: {
+  showVideoEffectsOptions: boolean;
+  setshowVideoEffectsOptions: (showVideoEffectsOptions: boolean) => void;
+}): JSX.Element => {
+  return VideoEffectsPaneTrampoline(props.showVideoEffectsOptions, props.setshowVideoEffectsOptions);
+};
+
+const VideoEffectsPaneTrampoline = (
+  showVideoEffectsOptions: boolean,
+  setshowVideoEffectsOptions: (showVideoEffectsOptions: boolean) => void
```

Comment:
I think you can remove the boolean property showVideoEffectsOptions and this callback prop can just be
```
onDismiss: () => void
```
such that you can use the VideoEffectsPane component like so:
```
{
            /* @conditional-compile-remove(video-background-effects) */
            showVideoEffectsPane &&
            <VideoEffectsPane
              onDismiss={() => setShowVideoEffectsPane(false)}
            />
}
```